scripts/dev_scripts/integration_tests.sh

```diff
@@ -24,6 +25,11 @@ if [ $# -eq 3 ] && [ "$3" == "--update" ] ; then
     COMPARE_VSA="$COMPARE_VSA --update"
 fi
 
+function run_macaron() {
```

Comment:
Nit: Can we name this something a bit more descriptive? How about `clean_run_macaron` or something similar?